android/settings.gradle

```diff
@@ -6,7 +6,8 @@ rootProject.name = 'native'
 include ':app'
 
 include ':backpack-react-native'
-project(':backpack-react-native').projectDir = new File(rootProject.projectDir, '../node_modules/backpack-react-native/android')
+project(':backpack-react-native').projectDir = new File(rootProject.projectDir, '../lib/android')
```

Comment:
Moved to `lib/android` because using from `node_modules` causes issues with `gradle`. This happens because we install the package from the file system `file:./lib/backpack-react-native` and the way npm installs it, is by creating a link. This causes `gradle` to compile the files twice and it trows a `duplicate` reference error when compiling the tests.